src/services/mondomarket/index.jsx

```
@@ -391,93 +388,94 @@ export const MondoDetailsViewer = ({
+              navigation={true}
+              pagination={{
+                clickable: true,
+              }}
+              modules={[Navigation, A11y, Autoplay, Pagination]}
+              className="h-[100vh] w-full px-3"
+            >
+              {mondo?.images?.map((image, index) => (
+                <SwiperSlide key={index}>
+                  <img
+                    src={image?.image}
+                    alt="Mondo Image"
```

Biome (1.9.4)

[error] Avoid the words "image", "picture", or "photo" in img element alt text.

Screen readers announce img elements as "images", so it is not necessary to redeclare this in alternative text.

(lint/a11y/noRedundantAlt)